modules/apps/27-interchain-accounts/controller/keeper/grpc_query_test.go

```diff
@@ -12,33 +12,33 @@ func (suite *KeeperTestSuite) TestQueryInterchainAccount() {
 	testCases := []struct {
 		name     string
 		malleate func()
-		expPass  bool
+		errMsg   string
```

Comment:
Is it not possible to use `expErr` here?

Reply:
I wasn't able to. We return errors with `status.Errorf` which is not the same as wrapping errors. They are not a specific error type